pages/docs/subscription.en-US.mdx

```diff
@@ -0,0 +1,82 @@
+import { Callout } from 'nextra-theme-docs'
+
+# Subscription
+
+<Callout emoji="✅">
+  Please update to the latest version (≥ 2.1.0) to use this API.
+</Callout>
+
+## `useSWRSubscription`
+
+`useSWRSubscription` is a React hook that allows subscribing to real-time data sources with SWR. Here is the API documentation for the code you provided:
+
+```tsx
+useSWRSubscription(key: any, subscribe: Function): { data: any, error: Error | null }
+```
+
+### API
+
+This hook subscribes to a real-time data source using the subscribe function provided, and returns the latest data received and any errors encountered. The hook automatically updates the returned data as new events are received.
+
+#### Parameters
+
+- `key`: A unique key that identifies the data being subscribed to, same key as `useSWR` key.
+- `options`: accepts the following options
+  - `next`: callback function that is called when data is available or an error occurs.
+
+
+#### Return Values
+
+- `subscription`: An object with the following properties:
+  - `data`: The latest data received from the real-time data source.
+  - `error`: An Error object if an error occurred while subscribing to the real-time data source, otherwise null.
```

Comment:
```suggestion
  - `error`: An Error object if an error occurred while subscribing to the real-time data source, otherwise undefined.
```

Reply:
We should also mention that the error will be reset if new data is successfully received?